components/ErrorNotification/index.js

```diff
@@ -0,0 +1,13 @@
+import { notification } from "antd";
+
+export function notifyError(error) {
+  //This print is intentional and not a debug print
+  console.log(error);
```

Comment:
```suggestion
  console.error(error);
```
 How is this not a debug print? 😅